src/components/createPDF.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react'
+import React, { FormEvent, ReactElement } from 'react'
 import { Box } from '@material-ui/core'
 import { createPDFPopup } from '../pdfFiller/fill1040Fields'
+import { DonePagedFormProps } from './pager'
 
-export default function CreatePDF ({ navButtons }) {
-  const onSubmit = (e) => {
+export default function CreatePDF ({ navButtons }: DonePagedFormProps): ReactElement {
+  const onSubmit = (e: FormEvent<any>): void => {
     e.preventDefault()
-    createPDFPopup()
+    createPDFPopup().then(() => {}, () => {})
```

Comment:
Were these anonymous functions added for a specific reason?

Reply:
This is used to ignore the effects the promise returned by createPDFPopup(). This is a 'dangling future' I think it's called, since createPDFPopup returns a promise, and we need this function to simply return void. So the TS compiler catches this, but in this case there's not much to be done since the form's `onSubmit` call wants a synchronous function.

Reply:
Added code comment

Reply:
On second look this can be changed to
 ```
 const onSubmit = async (e: FormEvent<any>): Promise<void> => {
    e.preventDefault()
    await createPDFPopup()
  }
```
I think this will increase clarity